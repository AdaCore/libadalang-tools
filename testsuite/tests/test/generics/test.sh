gnattest -P pkg.gpr
gprbuild -q -P gnattest/harness/test_driver.gpr
gnattest/harness/test_runner
