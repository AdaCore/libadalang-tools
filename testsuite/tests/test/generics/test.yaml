description: |
  Generates a test harness for a generic package (instantiated in a regular
  package). One unimplemented test sould be generated.

driver: shell_script
