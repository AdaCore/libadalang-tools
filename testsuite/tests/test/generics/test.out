Units remaining: 2     Units remaining: 1     
my_generic.ads:4:4: instance at my_instance.ads:4:4: error: corresponding test FAILED: Test not implemented. (my_generic-test_data-tests.adb:44)
1 tests run: 0 passed; 1 failed; 0 crashed.
